packages/gerber-parser/lib/_parse-drill.js

```diff
@@ -10,6 +10,8 @@ var normalize = require('./normalize-coord')
 var parseCoord = require('./parse-coord')
 
 var RE_ALTIUM_HINT = /;FILE_FORMAT=(\d):(\d)/
+var RE_ALTIUM_PTH = /;TYPE=PLATED/
```

Comment:
Similar to other regexs in this file, the plated / non-plated logic could be contained in a single regex:

```js
RE_ALTIUM_PLATING_HINT = /;TYPE=(PLATED|NON_PLATED)/

// ...

} else if (RE_ALTIUM_PLATING_HINT.test(block)) {
  var platingMatch = block.match(RE_ALTIUM_PLATING_HINT)
  var drillPlating = platingMatch[1] === 'PLATED'
  parser._push(commands.set('drillPlating', drillPlating, line))
}

// ...
```